views/components/Drawer.jsx

```diff
@@ -127,9 +88,46 @@ const DrawerComponent = (props) => {
           </ListItemIcon>
           <ListItemText primary="Logout" />
         </ListItem>
+        <ListItem>
+          <Divider />
+        </ListItem>
       </List>
     </>
   )
 }
 
 export default DrawerComponent
+
+/**
+ *
+       <Typography noWrap={true} variant="subheading">
+        {props.user.name ? props.user.name : props.user.uid}
+      </Typography>
+      <table>
+        <tbody>
+          <tr>
+            <td>
+              <Typography noWrap={true}>{props.totalStudies}</Typography>
+            </td>
+            <td>
+              <Typography noWrap={true}>{props.totalSubjects}</Typography>
+            </td>
+            <td>
+              <Typography noWrap={true}>{props.totalDays}</Typography>
+            </td>
+          </tr>
+          <tr>
+            <td>
+              <Typography noWrap={true}>{'studies'}</Typography>
+            </td>
+            <td>
+              <Typography noWrap={true}>{'participants'}</Typography>
+            </td>
+            <td>
+              <Typography noWrap={true}>{'days'}</Typography>
+            </td>
+          </tr>
+        </tbody>
+      </table>
+      <Divider />
+ */
```

Comment:
```suggestion
```

Reply:
This file is removed from the merge [6b310d1](https://github.com/AMP-SCZ/dpdash/pull/458/commits/6b310d18b25129974d8ce470610170164ed5d426)